lib/core-integration/src/Test/Integration/Framework/DSL.hs

```diff
@@ -1017,6 +1019,18 @@ listByronWalletEp =
     , "v2/byron/wallets"
     )
 
+migrateByronWalletEp :: ApiByronWallet -> ApiWallet -> (Method, Text)
+migrateByronWalletEp wSrc wDest =
+    ( "POST"
+    , "v2/byron/wallets/" <> wSrc ^. walletId <> "/migrate/" <> wDest ^. walletId
+    )
+
+estimateByronWalletEp :: ApiByronWallet -> (Method, Text)
+estimateByronWalletEp w =
```

Comment:
Perhaps call this `getByronWalletMigrationInfoEp` (for consistency with the endpoint name) or `calculateByronMigrationCostEp`?

The name `estimateByronWalletEp` has two issues (as I can see):

1. It doesn't convey that it relates to migrations
2. It doesn't convey that the result is supposed to be exact.

Reply:
OK! `calculateByronMigrationCostEp` it is.